src/components/nav-user/nav-user.stories.ts

```diff
@@ -0,0 +1,67 @@
+import { boolean, storyFilters } from "../../../.storybook/helpers";
+import { text } from "@storybook/addon-knobs";
+import { html } from "../../../support/formatting";
+import readme from "./readme.md";
+import { placeholderImage } from "../../../.storybook/placeholderImage";
+
+export default {
+  title: "Components/Nav/Nav User",
+  parameters: {
+    notes: readme
+  },
+  ...storyFilters()
+};
+
+export const simple = (): string =>
```

Comment:
In order to keep these stories fully-focused on navigation user, would it be possible to:

* only use the nav user component here
* either update navigation stories to include an all-navigation-components story or a separate stories file to cover that?

Applies to other stories.

Reply:
Updated navigation component stories to utilize navigation-logo and navigation-user.